Project doesn't work for React

**Description**
The project doesn't work for React

**Reproduce**
Install the npm package into a new project following the README file

**Expected**
Everything work

**Actual**
Module not found: Error: Can't resolve './public/CS_Herobanner_Mainpage_mobile.svg' in 'C:\Users\Yanchev\Desktop\reacttest\node_modules\@infineon\design-system-bootstrap\src'
ERROR in ./node_modules/@infineon/design-system-bootstrap/src/index.scss (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[2]!./node_modules/resolve-url-loader/index.js??ruleSet[1].rules[1].oneOf[7].use[3]!./node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[4]!./node_modules/@infineon/design-system-bootstrap/src/index.scss) 30:37-107
Module not found: Error: Can't resolve './public/CS_Herobanner_Mainpage_mobile.svg' in 'C:\Users\Yanchev\Desktop\reacttest\node_modules\@infineon\design-system-bootstrap\src'

ERROR in ./node_modules/@infineon/design-system-bootstrap/src/index.scss (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[2]!./node_modules/resolve-url-loader/index.js??ruleSet[1].rules[1].oneOf[7].use[3]!./node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[4]!./node_modules/@infineon/design-system-bootstrap/src/index.scss) 31:37-100
Module not found: Error: Can't resolve './public/CS_Herobanner_Mainpage.svg' in 'C:\Users\Yanchev\Desktop\reacttest\node_modules\@infineon\design-system-bootstrap\src'
ruleSet[1].rules[1].oneOf[7].use[2]!./node_modules/resolve-url-loader/index.js??ruleSet[1].rules./node_modules/@infineon/design-system-bootstrap/src/index.scss) 33:37-106
Module not found: Error: Can't resolve './public/CS_Herobanner_Subpage_Mobile.svg' in 'C:\Users\Yanchev\Desktop\reacttest\node_modules\@infineon\design-system-bootstrap\src'

ERROR in ./node_modules/@infineon/design-system-bootstrap/src/index.scss (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[2]!./node_modules/resolve-url-loader/index.js??ruleSet[1].rules[1].oneOf[7].use[3]!./node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[4]!./node_modules/@infineon/design-system-bootstrap/src/index.scss) 34:37-100
Module not found: Error: Can't resolve './public/CS_Herobanner_Subpage1.svg' in 'C:\Users\Yanchev\Desktop\reacttest\node_modules\@infineon\design-system-bootstrap\src'

--- a/package.json
+++ b/package.json
@@ -15,7 +15,8 @@
     "dist",
     "/src/**/*.scss",
     "/src/**/*.js",
-    "index.js"
+    "index.js",
+    "/public/**"
   ],
   "source": "./dist/index.js",
   "sass": "./src/index.scss",

